src/js/SignInUI.js

```diff
@@ -0,0 +1,33 @@
+import AuthAPI from './AuthAPI';
+
+export default class SignIn{
+  static user = document.querySelector('#identifier');
+
+  static pass = document.querySelector('#login-pass');
+
+  static error = document.querySelector('#login-err');
+
+  static signIn() {
+    const { user, pass, error, errorLog} = SignIn;
+    if (user.value.length < 1 || pass.value.length < 1) {
+      errorLog(`There is an empty field`, error)
+    } else {
+      AuthAPI.login({
+        identifier: user.value,
+        password: pass.value,
+      })
+        .then(() => console.log('sign in')) // Передать даннные на UserUI
```

Comment:
what is the purpose of this `then` method?

Reply:
plug